controllers/storageclass/storageclass_controller.go

```diff
@@ -195,6 +195,54 @@ func (r *StorageClassWatcher) getFlashSystemClusterByStorageClass(sc *storagev1.
 	return foundCluster, fmt.Errorf(msg)
 }
 
+//lint:ignore U1000 Ignore unused function - planned for future use
+func (r *StorageClassWatcher) getManagementMapFromSecret(topologySecret *corev1.Secret) (map[string]v1alpha1.FlashSystemCluster, error) {
+	clustersByMgmtId := make(map[string]v1alpha1.FlashSystemCluster)
+	mgmtDataByMgmtId := make(map[string]interface{})
```

Comment:
I guess we use `interface` because this mapping contains values with different types (string/array)?
```"demo-management-id-1": {
    "username": "demo-username-1",
    "password": "demo-password-1",
    "management_address": "demo-management-address-1",
    "supported_topologies": [
      {
        "topology.block.csi.ibm.com/demo-region": "demo-region-1"
      }
    ]
  }
```

Reply:
Correct, The type interface{} applies to any value and that's why we need to use it here.